packages/paste-website/src/pages/icons/how-to-add-an-icon/index.mdx

```diff
@@ -82,7 +82,7 @@ Written for Sketch 59.1
 1. Copy and paste the Streamline icon onto your icon artboard.
 2. Select all the layers and make sure the border width is 1.
 3. Select the layer group. Lock the width-height aspect ratio. If the icon appears
-   larger than the bounds of the artboard, shrink the icon down so that it's visually aligned with our default text size. Use the "icon tester" artboard on the Icon tester page in the Sketch file to help. Resizing the icon to 16–20 px usually works.
+   larger than the bounds of the artboard, shrink the icon down so that it's visually aligned with our default text size. Use the "icon tester" artboard on the Icon tester page in the Sketch file to help. Resizing the icon to 16–18 px usually works.
 4. Duplicate the layer group. Rename the original group to "source" and hide it.
 5. Ungroup the new layer group. You’ll be working with these layers.
    ![Sample layer groups](../../assets/images/icon-guide/using-streamline-icons-3.png)
```

Comment:
```suggestion

```

Reply:
Not sure when the image broke, but we can just remove it.

Reply:
Probably when I moved this page. I think an extra `../` at the beginning will fix the path